specification/monitor/resource-manager/Microsoft.Insights/preview/2021-05-01-preview/diagnosticsSettingsCategories_API.json

```diff
@@ -0,0 +1,218 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get more information about diagnostic setting category",
+            "schema": {
+              "$ref": "#/definitions/DiagnosticSettingsCategoryResourceCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Gets the diagnostic setting": {
+            "$ref": "./examples/listDiagnosticSettingsCategories.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ProxyOnlyResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "A proxy only azure resource object"
+    },
+    "DiagnosticSettingsCategory": {
+      "properties": {
+        "categoryType": {
+          "enum": [
+            "Metrics",
+            "Logs"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "categoryType",
+            "modelAsString": true 
+          },
+          "x-nullable": false,
+          "description": "The type of the diagnostic settings category."
+        },
+        "categoryGroups": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "the collection of what category groups are supported."
+        }
+      },
+      "description": "The diagnostic settings Category."
+    },
+    "DiagnosticSettingsCategoryResource": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DiagnosticSettingsCategory",
+          "description": "The properties of a Diagnostic Settings Category."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/SystemData"
```

Comment:
>/SystemData [](http://example.com/codeflow?start=93&length=11)

I think it is case-sensitive. It should be systemData.

Reply:
fix